packages/adapter-auto/adapters.js

```diff
@@ -1,22 +1,28 @@
+// List of adapters to check for. `version` is used to pin the installed adapter version and should point
+// to the latest version of the adapter that is compatible with adapter-auto's current peerDependency version of SvelteKit.
 export const adapters = [
 	{
 		name: 'Vercel',
 		test: () => !!process.env.VERCEL,
-		module: '@sveltejs/adapter-vercel'
+		module: '@sveltejs/adapter-vercel',
+		version: '1'
```

Comment:
is there some way to get these automatically populated from the workspace? it seems like it's going to be really easy to overlook bumping these, which might cause more trouble that it solves

Reply:
We can't automatically bump them because the mechanism can't know if the bump warrants a major release of `adapter-auto` or not.
Example: 
- `adapter-vercel` releases new major but peerDep on SvelteKit stays the same -> should be a minor release in adapter-auto
- `adapter-vercel` releases a new major and bumps peerDep on SvelteKit -> should be a major release in adapter-auto

Reply:
Do we have the ability to specify semver ranges here of the other adapters? It might not be what's happening here, but I can imagine a situation where an adapter has a breaking change because of what options it accepts, but that change is irrelevant for people consuming it through the auto adapter because it doesn't let you specify any options anyway. Are we prepared for that sort of situation?

Reply:
I think we could just bump the adapter version here and it would be harmless, no? It's just a matter of remembering to do it

Reply:
Probably, yeah. I'm not sure what I was thinking about with the range. I think I had forgotten that the auto adapter doesn't itself have peer dependencies on the other adapters.